bin/lb-ng

```diff
@@ -50,9 +50,9 @@ function runGenerator() {
   // used to have a bug where it prevented the application from exiting.
   // To work around that issue, we are explicitly exiting here.
   //
-  // The exit is deferred to the next tick in order to prevent the Node bug:
-  // https://github.com/joyent/node/issues/3584
-  process.nextTick(function() {
+  // previously used process.nextTick, but in Node 6 is causing
+  // exit to happen before finish writing leading to missing data
+  process.stdout.once('drain', function() {
```

Comment:
- We need to support 0.10
- We also need to force the process  to exit. The reason for that is that some LoopBack connectors  open a connection pool at start, and this pool blocks the app from exiting "naturally".

> what about stderr?

Good point.